src/content/resources/faq.md

```diff
@@ -103,7 +103,7 @@ Dart chooses _familiarity_ over _excitement_.
 **Web or Flutter apps**
 : No, Dart doesn't support write to [web or Flutter apps][Flutter no mirrors].
 
-### Q. Can Dart add tuples, partial evaluation, ...?
+### Q. Can Dart add algebraic data types, union types, ...?"
```

Comment:
Let's revert this. I think the goal here is to help people who specifically are searching for tuple support and partial evaluation support.